unittests/snapshot_tests.cpp

```diff
@@ -808,4 +812,56 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_kv_snapshot, SNAPSHOT_SUITE, snapshot_suites)
    }
 }
 
+using next_t = eosio::producer_plugin::next_function<eosio::producer_plugin::snapshot_information>;
+
+void next_func(eosio::producer_plugin::snapshot_information& snap_info)
```

Comment:
I don't see next_func is used in this file.

Reply:
Good catch. I cleaned up the unused code.